### DISCUSSION TITLE
Domain name refused to recognize me?  Automatically?  HUH?

### DISCUSSION THREAD

**Discussion by SBBEnes** [community/community#40295] [GitHub Education]
### Select Topic Area

Bug

### Body

So it won't let me verify my account, automatically declines without any other option, because I used the SCHOOL domain, instead of the STUDENT domain.

You want me to offer you a bogus account with name@**students**.savannahtech.edu, instead of a real address of name@savannahtech.edu, because your database is set up incorrectly?  Without even a chance to respond? For this, or any school with a students subdomain, if you put in the school domain, it will automatically recognize the students subdomain.

![Untitled3](https://user-images.githubusercontent.com/119345891/204349959-04f4448c-8b15-4ba0-b073-f2ab4ec1fc22.jpg)

**Comment by ossanna16** [comment]
Hi there @SBBEnes and welcome to our community! Thank you for reporting this!
